testnet/tools/icos_deploy.sh

```diff
@@ -369,8 +369,8 @@ rm -rf "${TMPDIR}"
 echo "-------------------------------------------------------------------------------"
 cd "${REPO_ROOT}/testnet/ansible"
 
-echo "**** Remove eventual monitoring - ($(dateFromEpoch "$(date '+%s')"))"
-ansible ic_p8s_service_discovery_destroy.yml
+# echo "**** Remove eventual monitoring - ($(dateFromEpoch "$(date '+%s')"))"
+# ansible ic_p8s_service_discovery_destroy.yml
```

Comment:
Another accidental change? :smile:

Reply:
I am trying to get the OS images built, and thought maybe changing it from draft to non-draft makes a difference. I don't intend on merging this change, just get the image built.

Reply:
I should have included that in the description of the MR.

Reply:
OS images get built every time. They just don't get pushed to S3 *unless* there is a `[S3_UPLOAD]` in the MR title ([link](https://dfinity.slack.com/archives/CGZG5A18C/p1715861100906609?thread_ts=1715668536.711219&cid=CGZG5A18C)).

https://sourcegraph.com/github.com/dfinity/ic/-/blob/gitlab-ci/src/bazel-ci/main.sh?L25-29

`[RUN_ALL_BAZEL_TARGETS]` is not necessary, and causes more work to be done by the CI.

Reply:
I did set it to `[S3_UPLOAD]` but then I got this error: 

```suggestion
Run if [[ $CI_PULL_REQUEST_TITLE == *"[S3_UPLOAD]"* ]]; then
Error: The [S3_UPLOAD] flag has been deprecated. Please use [RUN_ALL_BAZEL_TARGETS] instead. It will automatically upload to S3.
Error: Process completed with exit code 1.
```

Reply:
oh yeah, they changed that :/ 

https://sourcegraph.com/github.com/dfinity/ic/-/blob/ci/bazel-scripts/main.sh?L16

Yep, you are right, now it's necessary to either have `[RUN_ALL_BAZEL_TARGETS]` in the PR title, or to target an `rc--*` branch.

Reply:
There should not be any difference in the disk image, there is difference in tests. But I haven't worked on this code for a while, so can't be sure. Best to ask IDX.